src/pytest_elasticsearch/factories.py

```diff
@@ -77,15 +97,44 @@ def elasticsearch_proc(executable='/usr/share/elasticsearch/bin/elasticsearch',
     :param str elasticsearch_logsdir: path for logs.
     :param elasticsearch_logsdir: path for elasticsearch logs
     """
+    commands_exec = {}
+    commands_exec['1.5'] = '''
+        {deamon} -p {pidfile}
+        --http.port={port} \
```

Comment:
multiline strings don't need lines broken like that?

Reply:
It's not for python, but for the shell command, we start the elasticsearch with.